petab/petab_schema.yaml

```diff
@@ -0,0 +1,46 @@
+# For syntax see: https://json-schema.org/understanding-json-schema/index.html
+#$schema: "https://json-schema.org/draft/2019-09/meta/core"
+$schema: "http://json-schema.org/draft-06/schema"
+description: PEtab parameter estimation problem config file schema
+
+properties:
+
+  petab_version:
+    type: string
+    description: Version of the PEtab format (e.g. 0.0.0a16)
+
+  parameter_file:
+    type: string
+    description: |
+      PEtab parameter file name containing parameters of all models listed in
+      `problems`. Absolute or relative path.
+
+  problems:
+    type: array
+    description: |
+      One or more (SBML model, measurement, condition files) tuples.
+    items:
+      type: object
+      description: PEtab SBML, condition and measurement tables
+      properties:
+        sbml_file:
+          type: string
+          description: PEtab SBML file
+        measurement_files:
```

Comment:
what is a use case for having multiple measurement files, but not multiple <rest>s?

Reply:
You may want to organize your measurements by origin of dataset and combine different subsets in different yaml files.
You are right though that the same argument may apply to condition tables too.
For SBML models, maybe as well. This will however require changes in a looooot of places, worth a separate PR.
I'd adapt the schema, so we don't need to change it too often, but raise NotYetImplemented.

Reply:
done